sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -0,0 +1,62 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+#from typing import Sequence
+#from collections import defaultdict
+#from datetime import datetime
+#from azure.eventhub import CheckpointStore, EventHubConsumerClient
+#from azure.data.tables import TableClient, TableServiceClient, UpdateMode
+#from azure.core.exceptions import ResourceModifiedError, ResourceExistsError, ResourceNotFoundError
```

Comment:
since we're not implementing anything in this PR, lets just remove these comments